Assets/Scripts/Movement/SlipTimeMover.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+
+namespace Movement
+{
+    public class SlipTimeMover : SlipTime.SlipTimeAdherent
```

Comment:
A thought: could we refactor SlipTimeAdherent to an interface (probably rename it to ISlipTimeAdherent) and then have this class extend Mover and implement ISlipTimeAdherent? As far as I know, C# allows getters and setters in interfaces, and using that approach might make things more compact. We might want to consider changing the methods in Mover to protected and then overriding them here if necessary.

Reply:
In theory, yes, that is the correct architecture. In practice, this is not possible because Unity requires a type that inherits from ‘Unity.Object’ (or something similar) in order to be able to look it up during the ‘Update()’ method. So unfortunately we can’t ask Unity to look up everything that implements an interface. This is at least what I determined in the limited research I did. Maybe there’s an alternate way I don’t know about.

Reply:
Boo. OK, sounds good.